Term Entry Lua Math cos() (#3451)
Codecademy/docs: content/lua/concepts/mathematical-library/terms/cos/cos.md

---
Title: 'cos()'

> Description: 'Evaluates the cosine of an angle represented in radians.'

Reviewer: ```suggestion
Description: 'Returns the cosine of an angle represented in radians.'
```

Subjects:
  - 'Computer Science'  
Tags:
  - 'Functions'
  - 'Math'
  - 'Numbers'
  - 'Arithmetic'
CatalogContent:
  - 'learn-lua'
  - 'paths/computer-science'
---

> The **`cos()`** function evaluates the cosine of an angle in radians.

Reviewer: ```suggestion
The **`cos()`** function returns the cosine of an angle given in radians.
```

## Syntax

```pseudo
$result = math.cos(angle);
```

`result`: The variable to store the output of the cosine operation.

> `math`: The standard Lua math library.
> `cos()`: The cosine function.

Reviewer: ```suggestion
```

> `angle`: The angle in radians for which you want to calculate the cosine.

Reviewer: ```suggestion
`angle`: The angle in radians.
```

## Example

The example below demonstrates the use of the `math.cos()` function to find the cosine of a 45-degree angle.

```lua
local angle = (math.pi / 4)  -- 45 degrees in radians
local cosineOutput = math.cos(angle)

print(cosineOutput)
```

This example will result in the following output:

```shell
0.70710678118655
```

> ## Codebyte Example

Reviewer: the Codecademy compiler doesn't currently support lua. Please make this a standard example or delete it

Author: Thank you for the edits @caupolicandiaz . What do you mean by a standard example..?

Reviewer: An example that is not a codebyte, which will include a separate ```shell block with the output
